apps/web/src/components/forms/password.tsx

```diff
@@ -18,12 +18,40 @@ import { useToast } from '@documenso/ui/primitives/use-toast';
 
 import { FormErrorMessage } from '../form/form-error-message';
 
+const commonPasswords = [
+  'password',
+  '12345678',
+  'password123',
+  'admin',
+  'qwerty',
+  'poiuyt',
+  'qwerty123',
+  'qwerty@123',
+];
```

Comment:
The majority of these passwords would fail the regex anyway. I think just remove common passwords for now.

Reply:
Okay, I'll remove the refine condition for the same as well